hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/health/RatisUnhealthyReplicationCheckHandler.java

```diff
@@ -31,18 +31,16 @@
 import static org.apache.hadoop.hdds.protocol.proto.HddsProtos.ReplicationType.RATIS;
 
 /**
- * This class handles UNHEALTHY RATIS replicas. Its responsibilities include:
- * <ul>
- *   <li>If only unhealthy replicas exist, check if there are replication
- *   factor number of replicas and queue them for under/over replication if
- *   needed.
- *   </li>
- *   <li>If there is a mixture of unhealthy and healthy containers this
- *   handler will only be called if there is no under or over replication after
- *   excluding the empty containers. If there is under or over replication the
- *   ECReplicationCheckHandler will take care of it first.
- *   </li>
- * </ul>
+ * This class handles RATIS containers which only have replicas in UNHEALTHY
+ * state, or CLOSED containers with replicas in QUASI_CLOSED state having
+ * less sequence ID. There are no other replicas. This class ensures that
+ * such containers have replication factor number of UNHEALTHY/QUASI_CLOSED
+ * replicas.
+ * <p>
```

Comment:
Nit: there is a stray <p> tag in the description.

Reply:
Added closing tag.